src/main/java/dk/bec/polonez/reservationsystem/service/ReservationService.java

```diff
@@ -62,6 +63,13 @@ public ResponseReservationDto addReservation(CreateReservationDto reservationDto
         Reservation savedReservation = reservationRepository.save(reservation);
         ResponseReservationDto.ResponseReservationDtoBuilder response = ResponseReservationDto.builder();
 
+        String email = savedReservation.getUser().getEmail();
```

Comment:
Czysto informacyjny komentarz :)

ale model, w ktorym z rezerwacji mozna przejsc do usera itp. czyli wlasciwie z kazdego obiektu do kazdego, koczy sie tym, ze zacieraja sie odpowiedzialnosci, miedzy obiektami/serwisami (a tak na prawde miedzy modulami, tutaj pewnie mozna wydzielic, modul rezerwacji, uzytkownikow, zarzadzania lokalami itp.). Podzielenie takiego kodu na niezalezne moduly lub w pozniejszym etapie na micro serwisy staje sie nie mozliwe <-- a to tylko oznaka, ze utrzymanie, dodawanie nowych featurow biznesowych staje sie z czasem coraz trudniejsze, a kod staje sie coraz mniej zrozumialy

Reply:
Dzięki, będę pamiętał na przyszłość :)